test/models/Stafftest.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file Defines tests for the Student sailsjs model.
+ */
+
+"use strict";
+const helperIntegrationTests = require("./_helpers");
+
+describe("Staff model", function() {
+    let testData = {
+        associations: {},
+        record: {}
+    };
+
+    let testStaff = {
+        username: "TESTSTUDENT@DEWV.EDU",
+        firstName: "TEST",
+        lastName: "STAFF",        
+        isSlpInstructor: "TESTSLP"
```

Comment:
You declared this as a boolean in the staff models but here you gave it a string.  Might want to fix this before pulling it in.